cms/djangoapps/contentstore/views/item.py

```diff
@@ -867,17 +867,23 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
         "user_partitions": get_user_partition_info(xblock, course=course),
     }
 
+    # update xblock_info with timed_exam information if the xblock category is sequential
+    if xblock.category == 'sequential':
+        xblock_info.update({
+            "is_time_limited": xblock.is_time_limited,
+            "default_time_limit_minutes": xblock.default_time_limit_minutes
+        })
+
     # update xblock_info with proctored_exam information if the feature flag is enabled
     if settings.FEATURES.get('ENABLE_PROCTORED_EXAMS'):
         if xblock.category == 'course':
             xblock_info.update({
-                "enable_proctored_exams": xblock.enable_proctored_exams
+                "enable_proctored_exams": xblock.enable_proctored_exams,
+                "enable_timed_exams": xblock.enable_timed_exams
```

Comment:
Why is enable_timed_exams behind the ENABLE_PROCTORED_EXAMS feature flag? I would expect it to be added up at line 875.